docs/en/05_Contributing/01_Code.md

```diff
@@ -278,6 +278,13 @@ The [Node.js](https://nodejs.org) JavaScript runtime is the foundation of our cl
 The configuration for an npm package goes in `package.json`. You'll see one in the root directory of `framework`. As well as being used for defining dependencies and basic package information, the `package.json` file has some handy scripts.
 
 ```
+$ npm run bundle
+```
+
+This script comiles the SCSS in CSS, the javascript source files into their dist and, minifies and concatinates all the
```

Comment:
This doesn't do anything with CSS.

"This script generates the `lib` and `leftandmain` JavaScript bundles"